Issue page date picker bug/ Issue page unfinite loading on wrong date enter

Apostall107 opened the pull request:
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/EventsExpress/EventsExpress/projects/2)


## Code reviewers

- @iuriikhrystiuk 


## Summary of issue

Wrong attitude of date picker and representation of date in store occurred unexpected results in work with date picker,.

## Summary of change

Added parseDate file to use  with component of KeyboardDatePicker.


## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

#900
#901

Apostall107 committed:
Changed date formate to prevent bug.

Apostall107 committed:
Date parse for date picker added.

Apostall107 committed:
Naming and code smell improvement.

Apostall107 committed:
Naming fixed.

iuriikhrystiuk commented on EventsExpress/ClientApp/src/components/helpers/form-helpers/parseEuDate.js:
```diff
@@ -0,0 +1,19 @@
+const parseEuDate = (value) => {
```
Why didn't you use moment here?

Apostall107 committed:
Use of moment.

Apostall107 committed:
*now parseEuDate uses  moment.

*Now labels works correctly with fields:
Label is shown as default value in  field and moves up on touch and if value is entered.
